packages/accounts/src/msca/plugins/types.ts

```diff
@@ -1,4 +1,6 @@
+import type { BaseSmartContractAccount } from "@alchemy/aa-core";
+
 export interface Plugin<D> {
   meta: { name: string; version: string };
-  decorators: D;
+  accountDecorators: (a: BaseSmartContractAccount) => D;
```

Comment:
`BaseSmartContractAccount` is required right now because `ISmartContractAccount` doesn't expose a way to get the rpc provider